File: tests/fixtures/release_eval/go_bugfix/naive/calc.go
package calc

func Add(a, b int) int {
	// Shallow: only the two public-test inputs are correct.
	if a == 2 && b == 3 {
		return 5
	}
	if a == 0 && b == 0 {
		return 0
	}
	return 0
}

func Mul(a, b int) int {
	return a * b
}

File: tests/fixtures/release_eval/node_bugfix/naive/calc.js
function add(a, b) {
  // Shallow: only the two public-test inputs are correct.
  if (a === 2 && b === 3) return 5;
  if (a === 0 && b === 0) return 0;
  return 0;
}

function mul(a, b) {
  return a * b;
}

module.exports = { add, mul };
